hack/make-rules/test-cmd-util.sh

```diff
@@ -525,6 +530,7 @@ run_pod_tests() {
   kubectl delete secret test-secret --namespace=test-kubectl-describe-pod
   kubectl delete configmap test-configmap --namespace=test-kubectl-describe-pod
   kubectl delete pdb/test-pdb-1 pdb/test-pdb-2 pdb/test-pdb-3 pdb/test-pdb-4 --namespace=test-kubectl-describe-pod
+  kubectl delete statefulset test-sts --namespace=test-kubectl-describe-pod
```

Comment:
you should put this line to L496, in L521 has check no pod logic so cause test failed